rules/mkdir.go

```diff
@@ -0,0 +1,32 @@
+package rules
+
+import (
+	"github.com/jessevdk/go-flags"
+	"github.com/nopecmd/nope/match"
+	"github.com/nopecmd/nope/models"
+	"github.com/nopecmd/nope/parse"
+	"strings"
+)
+
+func isMatchMkdir(cmd models.Command) bool {
+	return cmd.BaseCommand == mkdirBaseCommand
+}
+
+func getUndoMkdir(cmd models.Command) (string, error) {
+	var mkdirFlags struct {
+		Mode         string `short:"m"`
+		CreateInterm bool   `short:"p"`
+		Verbose      bool   `short:"v"`
+	}
+	filteredTokens, err := flags.ParseArgs(&mkdirFlags, cmd.TokensWithoutBase)
+	if err != nil {
+		return "", err
+	}
+	var dirPaths = parse.GetFilePathsFromTokens(filteredTokens)
+
+	return rmBaseCommand + " -rf " + strings.Join(dirPaths, " "), nil
```

Comment:
Why is rmdir better?

We need force if they attach some permissions to their newly made directory, I mean I can parse that out (it's the `m` flag), but having -f always will work